superchain/configs/sepolia-dev-0/semver.yaml

```diff
@@ -0,0 +1,7 @@
+l1_cross_domain_messenger: 1.6.0
+l1_erc721_bridge: 1.4.0
+l1_standard_bridge: 1.4.0
+l2_output_oracle: 1.5.0
+optimism_mintable_erc20_factory: 1.4.0
+optimism_portal: 1.9.0
+system_config: 1.7.0
```

Comment:
Note: I set these to the Base implementation versions. Not sure if these are reserved for Superchain contracts.

```
❯ cast call -r https://ethereum-sepolia-rpc.allthatnode.com 0x2cbD403d5BA3949D24ee4dF57805eaC612C2662f "version() (string)"
"1.6.0"

❯ cast call -r https://ethereum-sepolia-rpc.allthatnode.com 0xc3016ED03E087d092d576B585F5222fFD9cadc10 "version() (string)"
"1.3.0"

❯ cast call -r https://ethereum-sepolia-rpc.allthatnode.com 0x5638e55db5Fcf7A58df525F1098E8569C8DbA80c "version() (string)"
"1.4.0"

❯ cast call -r https://ethereum-sepolia-rpc.allthatnode.com 0xB5901509329307E3f910f333Fa9C4B4A8EE7CE1A "version() (string)"
"1.5.0"

❯ cast call -r https://ethereum-sepolia-rpc.allthatnode.com 0xEAa11178375e6B1078d815d6F9F85cBbb69b09Cd "version() (string)"
"1.4.0"

❯ cast call -r https://ethereum-sepolia-rpc.allthatnode.com 0x579c82A835B884336B632eeBeCC78FA08D3291Ec "version() (string)"
"1.9.0"

❯ cast call -r https://ethereum-sepolia-rpc.allthatnode.com 0x7f67dc4959cb3e532b10a99f41bdd906c46fdfde "version() (string)"
"1.7.0"
```

Reply:
we can follow up later with this, it's not critical for node superchain config.